### PR TITLE
start moving emails to messages

### PR DESCRIPTION
**Pull request by Baggerone**
This PR is now ready for a full review.

### CODE REVIEW

File: application/listeners/claim.go [hunk 1]
```diff
@@ -43,20 +44,8 @@ func claimReview1(e events.Event) {
...
-	msg.Template = domain.MessageTemplateClaimReview1Steward
-	msg.Data["memberName"] = memberName
-	msg.Subject = "Action Required. " + memberName + " just (re)submitted a claim for approval"
-
 	notifiers := getNotifiersFromEventPayload(e.Payload)
-	if err := notifications.Send(msg, notifiers...); err != nil {
-		domain.ErrLogger.Printf("error sending claim review1 notification, %s", err)
-	}
-
+	messages.ClaimReview1(claim, notifiers)
```
Comment (jason-jackson) [inline application/listeners/claim.go:48]: I think there should be an action. Either `messages.ClaimReview1(claim, notifiers).Send()` or `messages.SendClaimReview1(claim, notifiers)` or 
```
msg := messages.ClaimReview1(claim, notifiers)
notifications.SendMessage(msg)
```

File: application/messages/claim.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+)
+
+func addMessageClaimData(msg *notifications.Message, claim models.Claim) {
+	msg.Data["claimURL"] = fmt.Sprintf("%s/%s/%s", domain.Env.UIURL, domain.TypeClaim, claim.ID)
+	msg.Data["claimRefNum"] = claim.ReferenceNumber
+	return
+}
+
+func newClaimMessageForMember(claim models.Claim, member models.User) notifications.Message {
+	msg := notifications.NewEmailMessage()
```
Comment (jason-jackson) [inline application/messages/claim.go:18]: Should this be here instead of in notifications?

**Review by Schparky — APPROVED** [review]

File: application/messages/claim.go (hunk 2 above)
Comment (Baggerone) [reply application/messages/claim.go:18]: Yep. Good catch.

File: application/listeners/claim.go (hunk 1 above)
Comment (Baggerone) [reply application/listeners/claim.go:48]: Right. Steve had mentioned that too.

File: application/messages/claim.go (hunk 2 above)
Comment (Baggerone) [reply application/messages/claim.go:18]: Actually. Probably not. 
`notifications` has its `Message` struct, which this is using.  So this function may as well stay with it, I think.

**Review by Schparky — APPROVED** [review]

**Pull request merged**